lib/detect-testing-library-utils.ts

```diff
@@ -173,6 +174,69 @@ export function detectTestingLibraryUtils<
       return isNodeComingFromTestingLibrary(referenceNodeIdentifier);
     }
 
+    /**
+     * Determines whether a given node is a simulate event util related to
+     * Testing Library or not.
+     *
+     * In order to determine this, the node must match:
+     * - indicated simulate event name: fireEvent or userEvent
+     * - imported from valid Testing Library module (depends on Aggressive
+     * Reporting)
+     *
+     */
+    function isTestingLibrarySimulateEventUtil(
```

Comment:
Extracting generic logic from `isFireEventMethod` so I can reuse it for `isUserEventMethod`, which hasn't been implemented here yet but will be in the next PR. I thought I'd need it here but then I realized I wanted `isUserEventUtil` instead of `isUserEventMethod`.